lib/x86/setup.c

```diff
@@ -97,6 +101,46 @@ void find_highmem(void)
 		phys_alloc_init(best_start, best_end - best_start);
 	}
 }
+
+extern phys_addr_t ring0stacktop;
+
+unsigned long setup_tss(void);
```

Comment:
Can this line be deleted?

Reply:
I moved it to a header file. Function prototypes are requires because of the `-Wstrict-prototypes` and `-Werror` flags.

Reply:
The thing that confused me is that on line 107, we have the prototype, and then directly below, online 110, we have the function definition. I thought a function definition could also act as a prototype, thus making the prototype not needed.